app/ui-react/packages/ui/src/Layout/AppLayout.tsx

```diff
@@ -73,10 +67,6 @@ export const AppLayout: React.FunctionComponent<ILayoutBase> = ({
     : onNavigationExpand;
 
   const [breadcrumb, setHasBreadcrumb] = React.useState(null);
-  const [availableApps, setHasAvailableApps] = React.useState<
```

Comment:
I'm guessing we no longer need this? (ref #8561)

Reply:
Good catch! I'll remove it.

Reply:
Oops, just realized I read your comment/question wrong - you are correct, the cross console navigation effort was sunset so basically just needed to rip that stuff out and put it back like it was. Most of the discussion about it was offline, but here's an [issue](https://github.com/rh-uxd/appservices-patternfly/issues/53) that confirms it in case it's helpful .